doc/docs/overview.md

```diff
@@ -16,66 +20,106 @@ Opens a dialog with localized links to partner websites to purchase the supporte
 
 ### Load trace file...
 
-Opens file explorer and allows you to select a trace file. Traces captured using **Cellular Monitor** have the file extension `.mtrace`. You can also open files from `nrfutil trace` and legacy **Trace Collector** `.bin` files.
+Opens File Explorer and allows you to select a trace file. Traces captured using Cellular Monitor have the file extension `.mtrace`. You can also open files from `nrfutil trace` and `.bin` files from the legacy [nRF Connect Trace Collector](https://docs.nordicsemi.com/bundle/nrf-connect-tracecollector/page/index.html).
 
 ### Open trace file in Wireshark...
 
 Opens file explorer and allows you to select an `.mtrace` or `.bin` file. The selected file is converted to Packet Capture Next Generation (PcapNG) format and displayed in Wireshark.
 
 ## After selection
 
-When a device is selected, **Cellular Monitor** tries to discover its capabilities. The side panel options are updated depending on the results.
+When a device is selected, Cellular Monitor tries to discover its capabilities. The side panel options are updated depending on the results.
+
+![Cellular Monitor application window after selecting a device](./screenshots/cel_mon_overview.png "Cellular Monitor application window after selecting a device")
 
 ### Start
 
-Starts tracing.
+Starts tracing for the selected device.
+
+!!! tip "Tip"
+      You must have [Save trace file to disk](#save-trace-file-to-disk) toggled on to be able to start tracing.
```

Comment:
You can either have Save trace file to disk and/or Open in wireshark toggled in order to start a trace.